python/IECoreMaya/SceneShapeUI.py

```diff
@@ -383,9 +383,11 @@ def __selectedSceneShapes() :
 def __componentCallback( sceneShape, *unused ) :
 
 	parent = maya.cmds.listRelatives( sceneShape, parent=True, fullPath=True )[0]
-	maya.cmds.selectType( ocm=True, alc=False, facet=True )
+	maya.cmds.selectType( objectComponent=True, allComponents=False, facet=True )
 	maya.cmds.hilite( parent )
 
+	maya.cmds.select( clear=True )
```

Comment:
Out of curiosity, what bug did this fix?

Reply:
Good question - I remember that I had to do this, probably in order to not mess up the state in which the wireframes are drawn. I will go back and see if this is needed once we have selection working again.